marrow/mailer/release.py

```diff
@@ -4,11 +4,25 @@
 
 from collections import namedtuple
 
+VersionInfo = namedtuple(
+	'Version', (
+		'major',
+		'minor',
+		'micro',
+	),
+)
 
-version_info = namedtuple('version_info', ('major', 'minor', 'micro', 'releaselevel', 'serial'))(4, 1, 3, 'dev', 0)
-version = ".".join([str(i) for i in version_info[:3]]) + ((version_info.releaselevel[0] + str(version_info.serial)) if version_info.releaselevel != 'final' else '')
+AuthorInfo = namedtuple(
+	'Author', (
+		'name',
+		'email',
+	),
+)
 
-author = namedtuple('Author', ['name', 'email'])("Alice Bevan-McGregor", 'alice@gothcandy.com')
+info = VersionInfo(4, 2, 0)
+version = f"{info.major}.{info.minor}.{info.micro}"
```

Comment:
I am unsure how to do versioning for this repository properly, would appreciate all the opinions. :)